pkg/kubecost/kubecost_codecs.go

```diff
@@ -545,6 +572,34 @@ func (target *AllocationProperties) UnmarshalBinary(data []byte) (err error) {
 	target.Annotations = AllocationAnnotations(t)
 	// --- [end][read][alias](AllocationAnnotations) ---
 
+	if buff.ReadUInt8() == uint8(0) {
+		target.PVBreakdown = nil
+	} else {
+		// --- [begin][read][map](map[string]PVUsage) ---
+		bb := buff.ReadInt() // map len
+		aa := make(map[string]PVUsage, bb)
+		for jj := 0; jj < bb; jj++ {
+			var vvv string
+			cc := buff.ReadString() // read string
+			vvv = cc
+
+			// --- [begin][read][struct](PVUsage) ---
+			dd := &PVUsage{}
+			ee := buff.ReadInt()     // byte array length
+			ff := buff.ReadBytes(ee) // byte array
+			errA := dd.UnmarshalBinary(ff)
+			if errA != nil {
+				return errA
+			}
+			zzz := *dd
```

Comment:
Had to manually change = to := here

Reply:
That's not good. We'll have to see what's up, but manually adjusting the codec is a red flag.

Reply:
cc @mbolt35